tree/ntuple/v7/inc/ROOT/RNTupleReader.hxx

```diff
@@ -291,12 +291,24 @@ public:
       return GetView<T>(RetrieveFieldId(fieldName), objPtr);
    }
 
+   template <typename T>
+   RNTupleView<T> GetView(std::string_view fieldName, std::shared_ptr<T> objPtr, std::string_view typeName)
```

Comment:
Any specific reason for not making `typeName` a defaulted param in the existing `GetView`?